jvm/src/main/kotlin/app/cash/security_sdk/Certificate.kt

```diff
@@ -0,0 +1,65 @@
+package app.cash.security_sdk
+
+import app.cash.security_sdk.internal.providers.TrifleContentVerifierProvider
+import app.cash.security_sdk.internal.signers.TrifleContentSigner
+import okio.ByteString
+import org.bouncycastle.cert.X509CertificateHolder
+import org.bouncycastle.cert.X509v3CertificateBuilder
+import java.math.BigInteger
+import java.time.Instant
+import java.time.Period
+import java.util.Date
+
+/*
+ * Class representing a Trifle certificate.
+ *
+ * @property certificate - bytearray encoding of an x.509 certificate. Subject to change.
+ */
+class Certificate internal constructor(internal val certificate: ByteArray) {
```

Comment:
do we need this as a top level abstraction or should we just use the Certificate proto?

Reply:
Per our discussion last week (or so) following the API review, I think it makes sense for clients to be able to handle a Certificate object directly, and whether or not we decide to use a proto for serialization should be up to the serialize/deserialize logic.  In this particular case I've gone ahead and just gotten rid of the proto and directly written out the x509 certificate.  We can add proto wrapping to this thereafter if needed, but given that the current Certificate proto isn't actually protected (only the x509 cert is) we might just want to use x509 directly for now.

Reply:
> In this particular case I've gone ahead and just gotten rid of the proto
hmm I dont see public.proto updated

Reply:
ack: I'll update that